Makefile

```diff
@@ -1,5 +1,5 @@
 # setup default base image
-BASE_IMAGE ?= ruby:2.6-alpine
+BASE_IMAGE ?= ruby:3.0.0-alpine
```

Comment:
This change adds a risk in future that changes in percy common may not be compatible with renderer as renderer still does not use ruby 3

Reply:
reverted back